src/clj/fluree/db/api.cljc

```diff
@@ -244,11 +244,11 @@
 
   With the policy classes, finds all policies containing that class
   declaration."
-  ([db identity default-allow?]
-   (wrap-identity-policy db identity default-allow? nil))
-  ([db identity default-allow? values-map]
+  ([db identity]
+   (wrap-identity-policy db identity nil))
+  ([db identity values-map]
```

Comment:
I think removing `default-allow?` is a breaking change. What's the best way to manage it? (I'm very much in favor of removing it)

Reply:
Policy queries were just introduced and we are pre-release so I'm not concerned about it.